src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,19 +162,50 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   ])
   const isFilled = useMemo(() => isOrderFilled(order), [order])
   // Display isLowBalance warning only for active and partial fill orders
-  const isLowBalance = isOverBalance && !isUnlimited && !isFilled && !isScheduled && !isPendingOrder && !isExpiredOrder
+  const isLowBalance =
+    isOverBalance &&
+    !isUnlimited &&
+    !isFilled &&
+    !isScheduled &&
+    !isActiveNextBatch &&
+    !isPendingOrder &&
+    !isExpiredOrder
 
   const pending = useMemo(() => isPendingOrder && <PendingLink transactionHash={transactionHash} />, [
     isPendingOrder,
     transactionHash,
   ])
 
+  // Dima's trick to force component update
+  const [, forceUpdate] = useSafeState({})
+
+  // Updates the component every second, if it has one of the states where it should happen
+  useEffect(() => {
+    let timer: null | NodeJS.Timeout = null
+
+    if (isActiveNextBatch || isFirstActiveBatch) {
+      timer = setInterval(() => forceUpdate({}), 1000)
+    } else if (!!timer) {
+      clearTimeout(timer)
+    }
+
+    return (): void => {
+      if (timer) clearTimeout(timer)
```

Comment:
don't have to care about `if (timer)`, `clearTimeout(any nonsense)` is noop if not given a timer id
Don't even have to init `timer` as `null`

Reply:
Not to mention, should be `clearInterval`, :rofl:

Reply:
Let me tell you a secret. I first used `setTimeout` hihi